style.css

```diff
@@ -0,0 +1,105 @@
+/* stylelint-disable font-family-no-missing-generic-family-keyword */
+/* stylelint-disable length-zero-no-unit */
+/* stylelint-disable no-missing-end-of-source-newline */
+.container {
+  position: relative;
+  width: 375px;
+  height: 6715px;
+}
```

Comment:
- [OPTIONAL] kindly make sure you fix the padding and the margin of your body. there is an empty space after the flesh of your page